docs/HOWTO_INTEGRATE_TWILIO.md

```diff
@@ -39,3 +39,17 @@ If you follow the instructions above, every organization and campaign in your in
 - You can still set instance-wide credentials in the .env file (as described above). If you do, those credentials will be used as fallback if credentials aren't configured for an organization.
 - It is not required to configure all settings for all organizations. For example, to use a single site-wide Twilio account but with separate phone number pools for some organizations, follow the instuctions above and then set the Default Message Service SID (leaving the other fields blank) in the organizations settings for the orgs you want to override.
 - When using multiple Twilio accounts you will need to change the Inbound Request URL for your messaging service in the Twilio console [step 7 above]. It should look like `https://<YOUR_APP_URL>/twilio/<ORG_ID>` and `https://<YOUR_APP_URL>/twilio-message-report/<ORG_ID>`. The correct URL to use will be displayed on the settings page after you save the Twilio credentials.
+
+## Purchasing Numbers w/ Twilio
+- [How to set up Per-Org Number-Buying with Twilio](https://moveonorg.github.io/Spoke/#/HOWTO_BUY_NUMBERS_IN_SPOKE?id=how-to-set-up-per-org-number-buying-with-twilio)
+
+- How to choose area codes:
+    - press the plus button and a pop up menu will arise.
+    - Enter your desired area code
+    - Enter how many numbers you want to purchase
+    - Press submit
+
+- How to delete numbers:
+    - Only if you have clearance to purchase, you will have access to a delete button.
+    - However, this button deletes all numbers associated with a specific area code.
+    - If you want to delete a specific number, it's best done through Twilio.
```

Comment:
Move this section's `How to choose area codes` and `How to delete numbers` to [How to buy numbers through Spoke](https://moveonorg.github.io/Spoke/#/HOWTO_BUY_NUMBERS_IN_SPOKE?id=how-to-buy-numbers-for-message-services-in-spoke).


Under HowTo_integrate_twilio

Leave these links:
- How to Set up [Per Org](https://moveonorg.github.io/Spoke/#/HOWTO_BUY_NUMBERS_IN_SPOKE?id=how-to-set-up-per-org-number-buying-with-twilio) vs. 
- [Per Campaign](https://moveonorg.github.io/Spoke/#/HOWTO_BUY_NUMBERS_IN_SPOKE?id=how-to-set-up-per-campaign-number-buying-with-twilio) vs 
- [Twilio Direct](https://moveonorg.github.io/Spoke/#/HOWTO_BUY_NUMBERS_IN_SPOKE?id=how-to-buy-numbers-directly-in-twilio) number purchasing.